index.bs

```diff
@@ -861,9 +866,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. Let |options| be the value of <code>|options|.{{CredentialCreationOptions/publicKey}}</code>.
 
 1. If the {{PublicKeyCredentialCreationOptions/timeout}} member of |options| is [=present=], check if its value lies within a
-    reasonable range as defined by the platform and if not, correct it to the closest value lying within that range. Set a timer
+    reasonable range as defined by the [=client=] and if not, correct it to the closest value lying within that range. Set a timer
```

Comment:
might [=client=] be [=client platform=] ?

Reply:
Maybe... I figured platform conventions might play a role in choosing the range, but it's ultimately up to the client. I'm not against changing it; please confirm if you'd like me to make the change.